libpod/container_inspect.go

```diff
@@ -890,3 +890,25 @@ func (c *Container) generateInspectContainerHostConfig(ctrSpec *spec.Spec, named
 
 	return hostConfig, nil
 }
+
+func (c *Container) hostPidNS() (bool, error) {
```

Comment:
I read `hostPidNS()` like it's returning the host's PID NS.

```suggestion
// Return true if the container is running in the host's PID NS.
func (c *Container) inHostPidNS() (bool, error) {
```